publisher/guides/api-usage.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: How to use the Rest API
+---
+
+import YouTube from 'react-youtube';
+
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts in order to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password. 
+
+## WordPress application password
+
+This option is provided directly by WordPress (since version 5.6.). The application password is directly assigned to a user and can not be used for an
+interactive login. A user can create multiple application passwords. It is recommended to assign a separate application password for each client 
+application. If a client application was compromised, the user can revoke the password specifically for this application.
+
+### Create an application password
+
+<img src="/assets/publisher/api-users.png" />
+
+WordPress admins can create Application Passwords for each user, users can also create Application Passwords for their account. Access is through 
+the user profile page.
```

Comment:
WordPress admins can create Application Passwords for each user, users can also create Application Passwords for their account in the users profile page.

Reply:
Habe ich geändert